az-digital/az_quickstart
# Closes #2433 Add users to ManualMigrationLookup

## Description
Add the ability to migrate content programmatically that depends on a manual migration of users.

## Related issues
Closes #2433 

## How to test
- Run az_migration on a site, and pass the site off to a QA team, or content editing team
- Find out that you need to run additional migrations, but the sites are now out of sync due to manual entry of content.
- Create a new custom migration that has a dependency on existing/already migrated users combined with manually added users.
- See that users are referenced correctly in you custom migration.

## Types of changes

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
